### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by dimsunv** [tiangolo/meinheld-gunicorn-docker#134] [Questions] [answered]
/usr/src/cryptopro/pycades_0.1.22769/stdafx.h:19:10: fatal error: Python.h: No such file or directory
#0 0.695    19 | #include <Python.h>

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile: 

Without any more details or the code that you are running it would be very difficult to understand what you are trying to do and what is happening. If you still need help, please create a new discussion.
